documentation/source/newsfragments/1410.bug.rst

```diff
@@ -0,0 +1 @@
+Logging of non-string messages with colored log output is now working.
```

Comment:
Isn't this true of uncolored output too?

Reply:
Yes. But that was already the case before.

Reply:
Ah yep, I see now.